ambari-web/app/controllers/wizard/step1_controller.js

```diff
@@ -194,11 +194,13 @@ App.WizardStep1Controller = Em.Controller.extend({
   onNetworkIssuesExist: function() {
     if (this.get('networkIssuesExist')) {
       this.get('content.stacks').forEach(function (stack) {
-        stack.setProperties({
-          usePublicRepo: false,
-          useLocalRepo: true
-        });
-        stack.cleanReposBaseUrls();
+       if(stack.get('useLocalRepo') != true){
```

Comment:
Better to check for `stack.get('useLocalRepo') !== true` or `!stack.get('useLocalRepo')` condition

Reply:
done